.github/workflows/health.yaml

```diff
@@ -50,6 +50,16 @@ on:
         default: false
         type: boolean
         required: false
+      upload_coverage:
+        description: Whether to upload the coverage to coveralls
+        default: true
+        type: boolean
+        required: false
+      test_web:
+        description: Whether to run `dart test -p chrome` for coverage
+        default: true
```

Comment:
perhaps `default: false`?

Reply:
I was thinking about that, not sure what the right default is. I can try `false`.